lexicons/com/atproto/identity/defs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.defs",
+  "defs": {
+    "atprotoIdentity": {
+      "type": "object",
+      "required": ["did"],
+      "properties": {
+        "did": { "type": "string", "format": "did" },
+        "handle": {
+          "type": "string",
+          "format": "handle",
+          "description": "The validated handle of the account; or 'handle.invalid' if the handle did not bi-directionally match the DID document."
+        },
+        "tombstoned": {
```

Comment:
Since we got rid of the `tombstone` event, I'm not sure we have "tombstone" anywhere in the spec (except in PLC).

Tombstone as a concept is weird to apply to some DID methods such as did:web. I wonder if it may make sense to change the name of this property to something more generalized like `doesNotResolve` or something 🤔